enterprise_gateway/services/processproxies/k8s.py

```diff
@@ -57,7 +57,8 @@ def get_container_status(self, iteration):
                 if pod_status == 'Running' and self.assigned_host == '':
                     # Pod is running, capture IP
                     self.assigned_ip = pod_info.status.pod_ip
-                    self.assigned_host = pod_info.status.host_ip
+                    self.assigned_host = pod_info.status.pod_ip
```

Comment:
I think we should be consistent across the container sub-classes and set `self.assigned_host` to `self.container_name`

Reply:
Yeah, i can't agree more